modules/nf-core/vcf2maf/main.nf

```diff
@@ -0,0 +1,64 @@
+process VCF2MAF {
+
+    tag "$meta.id"
+    label 'process_low'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "bioconda::vcf2maf=1.6.21 bioconda::ensembl-vep=106.1"
```

Comment:
there is a newer version available from a fork  since the vcf2maf devs haven't released in a while, it is in the umcrr channel though. I have been using it because of compatibility issues with newer VEP versions. A bunch of annotations get the lowest priority  otherwise. we could discuss how we would handle this. Or maybe comment on the issue again in the hopes a proper release is made. Other option would be to try and update the bioconda recipte ourselves. I also think the VEP versoin here is not the same as in the latest sarek release. 

Somewhat wondering if we should disable recalling from our side to avoid unwanted side effects

Reply:
Thanks for the info! I totally overlooked that the module uses this older version. I will ask the recipe maintainer to update the version. Otherwise, would it be a temporary solution to create an environment.yml with the right anaconda channel to be used here?

Reply:
https://github.com/mskcc/vcf2maf/issues/340

Reply:
Let's see what happens: https://github.com/bioconda/bioconda-recipes/pull/42370